### PR TITLE
Regenerate projects client with latest DPG generator

### LINKED ISSUE #25148: Generate LLC for Question Answering authoring client
Now that inference has GA'd, we need to produce the authoring client for Question Answering.

- [ ] Generate `QuestionAnsweringProjectClient` in a subnamespace e.g., `Azure.AI.Language.QuestionAnswering.Project` (exact name TBD).
  - In .NET's autorest.md, there's already a `batch` section commented out you can use. Just set the namespace separately, and also disable putting models in a separate namespace, which archboard for .NET wants by default.
- [ ] Add champion scenario(s) to README.md
- [ ] Add additional authoring samples to samples/ directory.
- [ ] Update documentation including MigrationGuide.md as needed.
- [ ] Add/record tests with suitable code coverage.

### LINKED ISSUE #29051: [QUERY] Cannot locate class "QuestionAnsweringProjectsClient" in NuGet package "Azure.AI.Language.QuestionAnswering"
### Library name and version

Azure.AI.Language.QuestionAnswering 1.1.0-beta.1

### Query/Question

Based on the documentation found [here](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering), in order to interact with the Question Answering service, I just need to create an instance of the [QuestionAnsweringProjectsClient](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/Generated/QuestionAnsweringProjectsClient.cs) for managing projects.

However, after adding the corresponding NuGet package [Azure.AI.Language.QuestionAnswering](https://www.nuget.org/packages/Azure.AI.Language.QuestionAnswering/1.1.0-beta.1), the mentioned class cannot be found.

The same applies to the current non-beta version.

Where can we find the [QuestionAnsweringProjectsClient](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/Generated/QuestionAnsweringProjectsClient.cs) class?

Thank you!

### Environment

Windows 11
.NET 6
Visual Studio 2022 17.2.2
Azure.AI.Language.QuestionAnswering 1.1.0-beta.1
Azure.AI.Language.QuestionAnswering 1.0.0

### LINKED ISSUE #29958: Switch Question Answering live tests over to CLU static test resources
QA is using the old Surface KB/FAQ and needs to switch over to the static test resources we use for sdk/cognitivelanugage - CLU, specifically. This means rewriting all the test inputs and re-recording.

Also re-enable the AAD support test, which will link to this issue.

Resolves ignored test in PR #29665

### PR DESCRIPTION
**Pull request by heaths**
Fixes #25148, fixes #29958, and resolves #29051

### CODE REVIEW

File: sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/CHANGELOG.md [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 ### Features Added
 
-- Added the `QuestionAnsweringProjectsClient` to manage Question Answering projects.
+- Added the `QuestionAnsweringAuthoringClient` to manage Question Answering projects.
```
Comment (kristapratico) [inline sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/CHANGELOG.md:23]: should this rename be under 1.1.0-beta.3?

File: sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ dotnet add package Azure.AI.Language.QuestionAnswering
 
 ### Authenticate the client
 
-In order to interact with the Question Answering service, you'll need to either create an instance of the [`QuestionAnsweringClient`][questionanswering_client_class] class for querying existing projects or an instance of the [`QuestionAnsweringProjectsClient`][questionansweringprojects_client_class] for managing projects within your resource. You will need an **endpoint**, and an **API key** to instantiate a client object. For more information regarding authenticating with Cognitive Services, see [Authenticate requests to Azure Cognitive Services][cognitive_auth].
+In order to interact with the Question Answering service, you'll need to either create an instance of the [`QuestionAnsweringClient`][questionanswering_client_class] class for querying existing projects or an instance of the [`QuestionAnsweringAuthoringClient`][questionansweringprojects_client_class] for managing projects within your resource. You will need an **endpoint**, and an **API key** to instantiate a client object. For more information regarding authenticating with Cognitive Services, see [Authenticate requests to Azure Cognitive Services][cognitive_auth].
```
Comment (kristapratico) [inline sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/README.md:26]:
```suggestion
In order to interact with the Question Answering service, you'll need to either create an instance of the [`QuestionAnsweringClient`][questionanswering_client_class] class for querying existing projects or an instance of the [`QuestionAnsweringAuthoringClient`][questionansweringauthoring_client_class] for managing projects within your resource. You will need an **endpoint**, and an **API key** to instantiate a client object. For more information regarding authenticating with Cognitive Services, see [Authenticate requests to Azure Cognitive Services][cognitive_auth].
```

Up to you, but I like to update the name of the link to match, too.

File: sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/README.md [hunk 3]
```diff
@@ -46,15 +52,21 @@ AzureKeyCredential credential = new AzureKeyCredential("{ApiKey}");
 QuestionAnsweringClient client = new QuestionAnsweringClient(endpoint, credential);
 ```
 
-#### Create a QuestionAnsweringProjectsClient
+#### Create a QuestionAnsweringAuthoringClient
+
+To use the `QuestionAnsweringAuthoringClient`, make sure you use the right namespace:
+
+```C# Snippet:QuestionAnsweringAuthoringClient_Namespace
+using Azure.AI.Language.QuestionAnswering.Projects;
```
Comment (kristapratico) [inline sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/README.md:60]:
```suggestion
using Azure.AI.Language.QuestionAnswering.Authoring;
```

**Review by kristapratico — APPROVED** [review]

File: sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/CHANGELOG.md (hunk 1 above)
Comment (heaths) [reply sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/CHANGELOG.md:23]: Good catch. Too broad with my find/replace-all.

File: sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/README.md (hunk 3 above)
Comment (heaths) [reply sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/README.md:60]: Thanks. All the markdown locations with code should be fixed when I'm done. We extract from code which is fixed.

Comment (heaths) [reply sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/README.md:60]: Mostly, I pushed this early for @AlexanderSher to help investigate Azure/autorest.csharp#2726.

**Pull request merged**